src/lib/y2partitioner/widgets/blk_devices_table.rb

```diff
@@ -281,15 +281,34 @@ def type_label(device)
 
         fs = filesystem(device)
 
-        if fs&.multidevice?
-          btrfs_multidevice_type_label(fs)
+        if device.journal?
+          journal_type_label(fs)
+        elsif show_multidevice_type_label?(fs)
+          multidevice_type_label(fs)
         elsif fs
           formatted_device_type_label(device, fs)
         else
           unformatted_device_type_label(device)
         end
       end
 
+      # Whether the "Part of *fs.type*" label should be displayed
+      #
+      # @note the Ext3/4 filesystem could be detected as a
+      # {BlkFilesystem#multidevice? multidevice} filesystem when its journal is
```

Comment:
NP: is this ` {BlkFilesystem#multidevice? multidevice}` correctly rendered by yard? I would simply say "... could be detected as a multi-device filesystem ..."

Reply:
> is this {BlkFilesystem#multidevice? multidevice} correctly rendered by yard?

Not, it isn't. But simply because the namespace is missing. `{Y2Storage::Filesystems::BlkFilesystem#multidevice? multidevice}` works as expected, 

![Screenshot_2019-10-01 Class Y2Partitioner Widgets BlkDevicesTable — Documentation by YARD 0 9 16(1)](https://user-images.githubusercontent.com/1691872/65998640-a456fb00-e493-11e9-870b-ba40d421b5e7.png)

However, as you can see in the screenshot above, the `@note` is not rendered as expected. It seems that the tag puts the note isn't useful for multi-line notes besides it 

> Adds an emphasized note **at the top of the docstring** for the object

So, I'll remove it 

![Screenshot_2019-10-01 Class Y2Partitioner Widgets BlkDevicesTable — Documentation by YARD 0 9 16(2)](https://user-images.githubusercontent.com/1691872/65999002-6d351980-e494-11e9-935a-d98f6d2140ca.png)

and also to apply your suggestion

![Screenshot_2019-10-01 Class Y2Partitioner Widgets BlkDevicesTable — Documentation by YARD 0 9 16(3)](https://user-images.githubusercontent.com/1691872/65999116-a66d8980-e494-11e9-8510-d4b9f5f18b7b.png)

Thanks.